package.json

```diff
@@ -23,10 +23,13 @@
     "lint:style": "wp-scripts lint-style",
     "lint": "npm-run-all lint:php lint:js lint:style",
     "package": "npm run build && grunt package",
-    "deploy": "npm run build && grunt deploy",
+    "deploy": "npm run build && grunt deploy && npm run release",
```

Comment:
Should it tag it before doing the WP.org SVN push? I think the SVN push looks for the latest tag, right? Or do we always push to `trunk` without the tags?

Reply:
We run the replacement of the version in the `package` task which is a dependency of `deploy`. I think that `wp-deploy` uses the plugin version for tagging.